firmware/thruna/BMS/CMakeLists.txt

```diff
@@ -50,6 +50,24 @@ list(REMOVE_ITEM GOOGLETEST_TEST_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/Test/Src/main
 set(GOOGLETEST_TEST_INCLUDE_DIRS
     "${CMAKE_CURRENT_SOURCE_DIR}/Test/Inc")
 
+    
+# INCLUDES and SOURCES from shared
+# Define the shared code that you want to use in this project
+
+
+set(SELECTED_SHARED_SRCS
+    ${ALL_SHARED_APP_SRCS}
+    ${ALL_SHARED_IO_SRCS}
+    ${ALL_SHARED_HW_SRCS}
+)
+
+set(SELECTED_SHARED_INCLUDE_DIRS
+    ${ALL_SHARED_APP_INCLUDE_DIRS}
+    ${ALL_SHARED_IO_INCLUDE_DIRS}
+    ${ALL_SHARED_HW_INCLUDE_DIRS}
+)
+
+
```

Comment:
nit: Can you make whitespace consistent on the board files? i.e. remove the double newlines